### ISSUE TITLE
VST3 + Cubase / Nuendo + setting an automation value manually

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/Dplug#357]
```
I found a bug in the VST3 version of Panagement 1.4 (the free windows version). When trying to manually change the value of an automation (inserting a specific number) in Nuendo 8 and in Cubase 9.5 the DAW crashes. The VST2 version and the previous version I had (1.2) didn't crash when I did it. I checked with someone else who uses this version in Nuendo 8 and it happen to her too. BTW we both run Windows 7
```

**Comment by p0nce** [comment]
Confirmed with Cubase 9.5 + Win, very easy to reproduce using any Dplug plugin and inserting a number here
![image](https://user-images.githubusercontent.com/1067485/61537017-5571fb80-aa36-11e9-848e-4e34f6ebc5d9.png)

**Comment by p0nce** [comment]
Fixed in v8.4.9, very humiliating bug (bad use of sscanf, then copy-paste of that code...)
